test/cppsuite/test_harness/timestamp_manager.h

```diff
@@ -65,14 +65,15 @@ class timestamp_manager : public component {
         wt_timestamp_t latest_ts_s;
 
         /* Timestamps are checked periodically. */
-        latest_ts_s = (_latest_ts >> 32);
+        latest_ts_s = (get_next_ts() >> 32);
```

Comment:
I made this change as in theory no-one could be calling get_next_ts, thus not bumping our _latest_ts, and stalling the timestamps.

Reply:
Then we should make get_next_ts private

Reply:
It's called in various other components.

Reply:
What's the reason for this bit shifting anyway?

Reply:
See line 101 in this file, but basically we mimic mongodb timestamp style with its counter and a bit shift.